pages/api/hello.ts

```diff
@@ -0,0 +1,13 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
```

Comment:
lets just remove this entire thing, unless we need a serverless api route for something